Mastonet/MastodonClient.Status.cs

```diff
@@ -147,7 +147,7 @@ public Task<MastodonList<Account>> GetFavouritedBy(string statusId, ArrayOptions
             data.Add(new KeyValuePair<string, string>("poll[expires_in]", poll.ExpiresIn.TotalSeconds.ToString()));
             if (poll.Multiple.HasValue)
             {
-                data.Add(new KeyValuePair<string, string>("poll[multiple]", poll.Multiple.Value.ToString()));
+                data.Add(new KeyValuePair<string, string>("poll[multiple]", poll.Multiple.Value.ToString().ToLowerInvariant()));
```

Comment:
Did you notice a bug if we send True/False without lowering ?
If so, maybe we should add this `.ToLowerInvariant() ` also on line 155 for HideTotals

Reply:
I did. It defaults to false when not lowering. And yeah, happens to HideTotals too. I'm gonna add it.